pubspec.yaml

```diff
@@ -7,18 +7,18 @@ environment:
   sdk: '>=2.12.0-0 <3.0.0'
 
 dependencies:
-  analyzer: ^1.0.0
-  archive: ^3.0.0
-  args: ^2.0.0
-  dart_style: ^2.0.0
-  http: ^0.13.0
+  analyzer: ^2.2.0
+  archive: ^3.1.2
```

Comment:
I'm not sure if we should raise the minimum version of all other dependencies (e.g. `archive`, `args`, etc.).
I believe it is better to have a more flexible package than the one that works only with the latest dependencies.

In case I've missed something, or you really need the latest versions of all other packages, please let me know.

Reply:
yep you right, only major version should be updated.